example/docker-compose.extra.yml

```diff
@@ -62,25 +69,26 @@ services:
     networks:
       default:
         aliases:
-         - wdqs.svc
+         - ${WDQS_HOST}
     environment:
+      - VIRTUAL_HOST=${WDQS_HOST}
+      - VIRTUAL_PORT=${WDQS_PORT}
       - WIKIBASE_HOST
-      - WDQS_HOST=wdqs.svc
-      - WDQS_PORT=9999
-    expose:
```

Comment:
Note: The `expose` key on a Docker Compose service is mostly just a notation, and has no practical effect in our setup. The port 9999 is configured for the WDQS service, and available on that container regardless of the `expose` value. All ports on a container are by default available to other nodes in the Docker network.

Reply:
Yes, it is always just a documentation feature. We were not using the key for any of the other services (Elasticsearch, WDQS proxy, etc) so think it's better to take it out or notate it on all the relevant services for consistency. I think we should just leave it out as I think its misleading and becomes the only param in our Docker compose setup that has no purpose other than documentation. 

This from ChatGPT is helpful context as well, I suspect this related to why it was there I the first place:

> Me: Was "expose" used in earlier versions of Docker compose in a different way such that it was more than documentation and had additional practice effects?
> 
> ChatGPT: In earlier versions of Docker Compose, specifically in versions 1 and 2, the expose key had a more significant impact compared to its role in version 3. In Docker Compose version 1 and 2, the expose key not only documented the ports but also implicitly mapped them to the host machine. This meant that if you used expose without specifying the ports key, the service's ports would be exposed on the host machine as well. 

Because of this you'll notice if you Google on this topic there is a lot of conflicting and I think often misleading information. For all these reasons I think we just don't "expose".